api/XMLFormProcessor.inc

```diff
@@ -209,16 +211,16 @@ class XMLFormProcessor {
   protected function cleanupNodeRegistry(FormElement $element) {
     // Checks if the form elements associated with a DOMNodes where
     // removed. If any found, verifies if the associated DOMNodes are
-    // attributes of an orphan node or orphan ones, removing those from
-    // nodeRegistry.
+    // attributes of an orphan node or descentants of one that had a delete
```

Comment:
Descendants. Only mentioning it because you significantly improved the spelling in the next lines. :)